lib/src/components/NxDivider/NxDivider.scss

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ .nx-divider {
```

Comment:
This file needs to be deleted

Reply:
> This file needs to be deleted

Check 1c8d806169e9f3fcf770a7d8dc42b9e9ab95436c